airflow-core/src/airflow/ui/public/i18n/locales/en/hitl.json

```
@@ -6,6 +6,7 @@
       "received": "Reviewed"
     }
   },
+  "needsReview": "Needs Review",
```

I understand that many use cases are probably "Review" kind of but the purpose of HITL is not only Approval, but also some "Decsion" or "Input Request".

As we anmed all previously "Action Required" or "Repsonse" which is neutral I'd favor not changing the terms here but keeping it neutral as "Needs Action". Also in the "Browse..." Menu we have it as "Required Actions"
```suggestion
  "<to be renamed>": "Requires Action",
```

---

I think @cmarteepants's idea here is that "Review" does in fact cover each type of hitl. It all requires manual human review+action. 

I was going to work on renaming the rest in a different PR. But let me refactor this one to reduce the i18n surface area and focus the next PR on how much we want to rename